usr.sbin/inetd/inetd.h

```diff
@@ -213,4 +213,8 @@ parse_v2_result	parse_syntax_v2(struct servtab *, char **);
 
 #define SERVTAB_COUNT_MAX (SIZE_MAX - (size_t)1)
 
+/* Standard logging and debug print format for a servtab */
+#define SERV_FMT "%s/%s"
+#define SERV_PARAMS(sep) sep->se_service,sep->se_proto
+
```

Comment:
These formats are now used any time a service name is printed.  SERV_FMT in the printf format string, SERV_PARAMS in the parameters.